packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -98,6 +123,16 @@ export class ProblemContribution extends AbstractViewContribution<ProblemWidget>
             label: 'Collapse All',
             order: '0'
         });
+        menus.registerMenuAction(ProblemsMenu.PROBLEMS, {
+            commandId: ProblemsCommands.COPY.id,
+            label: 'Copy',
+            order: '1'
```

Comment:
I believe the final order should be:
- Copy
- Copy Message
- Collapse All (possibly under its own subgroup)